internal/helpers/strings.go

```diff
@@ -0,0 +1,24 @@
+package helpers
```

Comment:
I would probably not go for generic helpers packages but rather something string related (like e.g. `package str` or `package stringz`)